templates/repo/issue/view_content/pull.tmpl

```diff
@@ -364,6 +373,14 @@
 						<i class="icon icon-octicon">{{svg "octicon-x"}}</i>
 					{{$.i18n.Tr "repo.pulls.blocked_by_outdated_branch"}}
 					</div>
+				{{else if .IsBlockedByChangedProtectedFiles}}
+					<div class="item text red">
+						<i class="icon icon-octicon">{{svg "octicon-x" 16}}</i>
+						{{$.i18n.Tr (TrN $.i18n.Lang $.ChangedProtectedFilesNum "repo.pulls.blocked_by_changed_protected_files_1" "repo.pulls.blocked_by_changed_protected_files_n") | Safe }}
+						{{range .ChangedProtectedFiles}}
```

Comment:
I think something like this would work better:

```						
<div class="ui ordered list">
	{{range .ChangedProtectedFiles}}
		<div data-value="-" class="item">{{.}}</div>
	{{end}}
</div>
```

![chrome_2020-09-15_14-43-37](https://user-images.githubusercontent.com/1447794/93212068-62859a00-f762-11ea-9f00-092f15daba7b.png)